release-notes/v1.2.0.yaml

```diff
@@ -0,0 +1,149 @@
+date: November 6, 2024
+
+# Changes that are expected to cause an incompatibility with previous versions, such as deletions or modifications to existing APIs.
+breaking changes: |
+  Gateway API GRPCRoute and ReferenceGrant v1alpha2 have been removed.
+  Please refer to the [Gateway API v1.2.0 documentation](https://github.com/kubernetes-sigs/gateway-api/releases) for more information.
+  Removed default CPU limit of the Envoy Gateway deployment
+  Changed default Envoy shutdown settings: drain strategy has been changed to immediate, default minDrainDuration, drainTimeout and terminationGracePeriodSeconds have been set to 10s, 60s and 360s respectively
+
+# New features or capabilities added in this release.
+new features: |
+  Added support for Gateway-API v1.2.0
+  Added support for IPv4/IPv6 Dual Stack for Envoy listeners and BackendRef resources
+  Added support for EG standalone(host deployment) mode (experimental)
+  Added support for JWT claims based Authorization in SecurityPolicy CRD
+  Added support for Direct Response in HTTPRouteFilter CRD
+  Added support for Response Override in BackendTrafficPolicy CRD
+  Added support for RequestTimeout in BackendTrafficPolicy CRD
+  Added support for inverting header matches for rate limit in BackendTrafficPolicy CRD
+  Added support for client TLS session resumption in ClientTrafficPolicy CRD
+  Added support for HTTPRouteFilter and path regex rewrite
+  Added support for host header rewrite in HTTPRouteFilter CRD
+  Added support for Listener Access Log in EnvoyProxy CRD
+  Added support for Datadog tracing support in EnvoyProxy CRD
+  Added support for request response sizes stats in EnvoyProxy CRD
+  Added support for modifying container SecurityContext for Envoy Gateway deployment in Helm
+  Added support for wildcard matching for CORS AllowMethods and AllowHeaders settings in SecurityPolicy CRD
+  Added support for match conditions for access log in EnvoyProxy CRD
+  Added support for using BackendCluster to represent OIDCProvider
+  Added support for RecomputeRoute for ExtAuth in SecurityPolicy CRD
+  Added support for sharing token cookies between multiple domains in SecurityPolicy CRD
+  Added support for JSONPatches for proxy bootstrap modifications in EnvoyProxy CRD
+  Added support for LB priority for non xRoute endpoints
```

Comment:
Lets change this to `Added support for Active Passive Failover Backends` so it includes both xRoute and non xRoute